tools/cli/commands/build.js

```diff
@@ -768,6 +770,35 @@ async function buildProject( t ) {
 		if ( composerJson.repositories.length === 0 ) {
 			delete composerJson.repositories;
 		}
+
+		// Update dependency version numbers in composer.json.
+		const composerDepTyes = [ 'require', 'require-dev' ];
+		for ( const key of composerDepTyes ) {
+			if ( composerJson[ key ] ) {
+				for ( const [ pkg ] of Object.entries( composerJson[ key ] ) ) {
+					for ( const ctxPkg of Object.values( t.ctx.versions ) ) {
+						if ( ctxPkg.name === pkg ) {
+							let massagedVer = ctxPkg.version;
+
+							// Truncate non-0.x 'require' package versions to be two components only.
+							if (
+								key === 'require' &&
+								t.project.startsWith( 'packages/' && massagedVer[ 0 ] !== '0' )
```

Comment:
Not sure what happened there, but
```suggestion
								t.project.startsWith( 'packages/' ) && massagedVer[ 0 ] !== '0'
```